examples/nlp/language_modeling/megatron_lm_ckpt_to_nemo.py

```diff
@@ -360,7 +360,7 @@ def convert(local_rank, rank, world_size, args):
     trainer = Trainer(devices=args.gpus_per_node, accelerator='gpu', num_nodes=num_nodes)
 
     # TODO: reach out to PTL For an API-safe local rank override
-    trainer.accelerator.training_type_plugin._local_rank = local_rank
+    # trainer.accelerator.training_type_plugin._local_rank = local_rank
```

Comment:
Is this not needed anymore? Let's delete the comment.

Reply:
deleted.